src/NuGetPackageVerifier/third-party.json

```diff
@@ -0,0 +1,74 @@
+﻿{
+  "prefixWhitelist": [
+    "Libuv"
+    "Microsoft.AspNetCore",
```

Comment:
Curious why some of these prefixes end with dot and not others `Microsoft.AspNetCore` vs `Microsoft.Extensions.`